content/docs/getting-started/futures.md

```diff
@@ -1,19 +1,72 @@
 ---
 title: "Futures"
-weight : 1030
+weight : 1020
 menu:
   docs:
     parent: getting_started
 ---
 
-Futures, hinted at earlier in the guide, are the building block used to manage
-asynchronous logic. They are the underlying asynchronous abstraction used by
-Tokio.
+Let's take a closer look at futures. Tokio is built on top of the [`futures`] crate
+and uses its runtime model. This allows it to interop with other libraries also using
+the [`futures`] crate.
 
-The future implementation is provided by the [`futures`] crate. However, for
-convenience, Tokio re-exports a number of the types.
+**Note**: This runtime model is very different than async libraries found in
+other languages. While, at a high level, APIs can look similar, the way code
+gets executed differs.
 
-# What Are Futures?
+We'll be taking a closer look at the runtime in the next section, but a basic
+understanding of the runtime is necessary to understand futures. To gain this
+understanding, we'll firs tlook at the synchronous model that Rust uses by default
+and see how this differs from Tokio's asynchronous model.
+
+# Synchronous Model
+
+First, let's talk briefly about the synchronous (or blocking) model. This is the
```

Comment:
Probably reword to:

> "The Rust [standard library] uses a synchronous (or blocking) model.

The section above has "we'll first look at the synchronous model that Rust uses by default", so it is repeated.